src/test/kotlin/blackjack/model/CardsTest.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class CardsTest {
+    @Test
+    fun `should return true when cards in hand have condition for BlackJack`() {
+        val cardsInHand = Cards(mutableListOf(Card(Rank.ACE, Suit.HEARTS), Card(Rank.JACK, Suit.DIAMONDS)))
+        assertThat(cardsInHand.hasBlackJack()).isTrue()
+    }
+
+    @Test
+    fun `should return false when cards in hand dont fulfil condition for BlackJack`() {
+        val cardsInHand = Cards(
+            mutableListOf(
+                Card(Rank.ACE, Suit.HEARTS),
+                Card(Rank.NINE, Suit.DIAMONDS),
+                Card(Rank.ACE, Suit.DIAMONDS)
+            )
+        )
+        assertThat(cardsInHand.hasBlackJack()).isFalse()
+    }
```

Comment:
nice test! 👍 

How about explaining it in a way that more directly reflects the real-world rule?

- 'If it's 21 points with exactly 2 cards, it's Blackjack.'
- 'Even if it's 21 points, it's not Blackjack unless there are exactly 2 cards.'